polygeist/lib/Dialect/Polygeist/Transforms/LoopInternalization.cpp

```diff
@@ -881,12 +923,15 @@ struct LoopInternalization
                          AliasAnalysis &aliasAnalysis, DataFlowSolver &solver);
 
   /// Analyze the memory accesses that should use shared memory and aggregate
-  /// which dimensions use the loop induction variable.
+  /// which local ids for dimensions that use the loop induction variable.
   /// For example given (assume 'i' is the loop IV):
-  ///   A[tx][ty][i], B[tx][i][ty]
-  /// The set collected would contain {0,1}.
-  std::set<unsigned> getDimsThatUseLoopIV(LoopLikeOpInterface loop,
-                                          DataFlowSolver &solver) const;
+  ///   A[tx][i] => local id of dimension 1 is used for second index.
+  ///   A[i][tx] => local id of dimension 1 is used for first index.
+  ///   A[i][ty] => local id of dimension 0 is used for first index.
+  ///   A[ty][i] => local id of dimension 0 is used for second index.
+  /// The set collected for A[i][tx] and B[ty][i] would contain {0,1}.
```

Comment:
{0,1} -> {1,0}

Reply:
it is a set, and set is ordered increasingly.